yesod-auth-oauth2.cabal

```diff
@@ -1,5 +1,5 @@
 name:            yesod-auth-oauth2
-version:         0.1.2
+version:         0.1.3
```

Comment:
Can you revert this? Version/release is a separate concern from any one PR/feature.